integrations/defectdojo-JSONCreatedUsingSemgrepAPI/transformFindingsToJSON.py

```diff
@@ -0,0 +1,93 @@
+import requests
+import json
+import os
+from uploadSemgrepJSONToDefectDojo import uploadToDefectDojo
+import sys
+# Configuration
+deployment_slug = "swati"
+BASE_URL = "https://semgrep.dev"
+findings_url = BASE_URL+"/api/v1/deployments/"+deployment_slug+"/findings"
+SEMGREP_APP_TOKEN = os.environ["SEMGREP_API_TOKEN"]
+OUTPUT_FILE = "semgrep_findings_formatted.json"
+dd_url= "http://localhost:8080"
+product_name = "Semgrep SAST"
+engagement_name = "Semgrep Scans"
+dd_token = os.getenv("DEFECT_DOJO_API_TOKEN")
```

Comment:
If this comes up as none/unset, you'll need a way to error out properly before just erroring on the upload.